server.go

```diff
@@ -1154,6 +912,42 @@ func (s *Server) processStreamingRPC(t transport.ServerTransport, stream *transp
 	return t.WriteStatus(ss.s, status.New(codes.OK, ""))
 }
 
+func (s *Server) processUnaryRPC(t transport.ServerTransport, stream *transport.Stream, srv *service, md *MethodDesc, trInfo *traceInfo) (err error) {
+	return s.processRPC(t, stream, func(ss *serverStream) error {
+		reply, err := md.Handler(srv.server, ss.ctx, ss.RecvMsg, s.opts.unaryInt)
+		if err != nil {
+			return err
+		}
+		if err := ss.SendMsg(reply); err != nil {
+			return err
+		}
+		return nil
+	}, trInfo)
+}
+
+func (s *Server) processStreamingRPC(t transport.ServerTransport, stream *transport.Stream, srv *service, sd *StreamDesc, trInfo *traceInfo) (err error) {
+	return s.processRPC(t, stream, func(ss *serverStream) error {
+		var (
+			err    error
+			server interface{}
+		)
+		if srv != nil {
+			server = srv.server
+		}
+		if s.opts.streamInt == nil {
+			err = sd.Handler(server, ss)
```

Comment:
Can be simplified:

`return sd.Handler(server, ss)`

delete local var `err`, remove "else", etc

Reply:
Done :joy_cat: